awx/ui_next/src/screens/Project/shared/ProjectForm.jsx

```diff
@@ -226,29 +284,29 @@ function ProjectForm(props) {
                   {
                     git: (
```

Comment:
No subform for manual SCM type? Results in kind of an odd empty form section:

<img width="1434" alt="Screen Shot 2019-11-04 at 1 49 58 PM" src="https://user-images.githubusercontent.com/410794/68148480-10e96d80-ff0a-11e9-9d22-5584009fe2fa.png">

Reply:
I plan on opening a separate PR for the Manual form section.